extensions/src/main/java/org/pentaho/platform/web/http/context/PentahoSystemReadyListener.java

```diff
@@ -32,7 +32,8 @@
 import org.pentaho.platform.engine.core.system.PentahoSessionHolder;
 import org.pentaho.platform.engine.core.system.PentahoSystem;
 import org.pentaho.platform.util.logging.Logger;
-
+import javax.jcr.Repository;
+import org.apache.jackrabbit.core.RepositoryImpl;
```

Comment:
Shouldn't cast to the Impl. jcrRepository implements org.apache.jackrabbit.api.JackrabbitRepository which is OK to cast to.